src/Components/UI/Input/Input.tsx

```diff
@@ -25,8 +92,24 @@ const Input = (props: any) => {
                 <h1>search</h1>
             </div>
             {props.header && <img className="search_logo" src={SEARCH} alt="search"/>}
-            <input ref={inputRef}  onBlurCapture={props.onBlur} type={props.type} id={props.id} placeholder={props.placeholder} name={props.name} onFocus={()=>{setLabel(props.label)}}
-                   onBlur={()=>{setLabel('')}}/>
+            <input onChange={(event) => event.target.value.length > 0 ? searchAll(event) : setShowDropDown(false)}
+                   ref={inputRef} onBlurCapture={props.onBlur} type={props.type} id={props.id}
+                   placeholder={props.placeholder} name={props.name} onFocus={() => {
+                setLabel(props.label)
+            }}
+                   onBlur={() => {
+                       setLabel('')
+                   }}/>
+            {showDropDown &&
+                <div className="search_dropdown">
+                    <ul>
+                        {renderedResults && renderedResults.map(((item: any, index: number) =>
+                                <NavLink className='link' to={`/${item.type}/${item.id}`}>
+                                    <li key={index}>{item.display}</li>
```

Comment:
if there is an option, give the key non index value. In this case you can give it the value of the ID.

Reply:
Fixed